src/frontend/src/utils/helper.util.ts

```diff
@@ -22,7 +22,15 @@ export const authHeader = () => {
 
 export const getItemFromStorage = (key: string) => {
   const item = window.localStorage.getItem(key);
-  return item ? JSON.parse(item) : null;
+  return item ? item : null;
+};
+
+export const getPIAIntakeFormFlagFromStorage = () => {
```

Comment:
I don't think we should write flag specific methods in the helper util. Please make it generic where you pass the flag name and get corresponding value.

Reply:
change the name and make it generic